src/Catty/PlayerEngine/Sensors/Audio/LoudnessSensor.swift

```diff
@@ -38,6 +38,9 @@ class LoudnessSensor: NSObject, DeviceSensor {
     }
     
     func convertToStandardized(rawValue: Double) -> Double {
+        if !(self.getAudioManager()?.loudnessAvailable())! {
+            return type(of: self).defaultRawValue
```

Comment:
A raw value should not be returned by this function since it expects a standardized value. These changes will not be necessary if you set the defaultRawValue according to Apple's minimum volume value (no noise):
https://developer.apple.com/documentation/avfoundation/avaudiorecorder/1387176-averagepower